flocker/control/httpapi.py

```diff
@@ -485,6 +490,49 @@ def saved(_):
         saving.addCallback(saved)
         return saving
 
+    @app.route("/configuration/containers/<name>", methods=['DELETE'])
+    @user_documentation(
+        """
+        Remove a container from the configuration.
+
+        This will lead to the container being stopped and not being
+        restarted again.
+        """,
+        examples=[
+            u"remove a container",
+            u"remove a container with unknown name",
+        ]
+    )
+    @structured(
+        inputSchema={
+            '$ref': '/v1/endpoints.json#/definitions/configuration_container'},
```

Comment:
:question: Is this actually required for a `DELETE` request? The tests will pass with an empty `inputSchema`. Also this is a question more broadly relating to how our JSON schemas are generated from the YAML files; specifying the input schema here would in one sense suggest that there ought to be a request body with the endpoint's defined required fields such as the container's host and image. Obviously we don't supply those and the API calls succeed on a `DELETE` without a request body and the `name` specified in the request path (which is entirely the correct behaviour I'd expect), but this is the case even with this input schema present, in which case I'm left wondering what purpose it serves here. So this is more a question of curiosity as to how this works.